doc/list.md

```diff
@@ -1 +1,195 @@
 # List
+
+A `List` looks similar to an [Array](Array.md) but it is a pure Lean inductive type that does not
```

Comment:
Lists are a basic type in functional languages. It feels very weird to introduce them by making a connection with arrays.
I prefer you use the more standard style used by F#.
https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/lists

Reply:
Got it, I've changed the intro accordingly.